### PR TITLE
Past court dates #805

### LINKED ISSUE #805: past court dates should appear with casa_case details
Part of epic #880 // dependent on #807 

**What type of user is this for? [ volunteer/supervisor/admin/all or All CASA Admin]**
all

**Where does/should this occur?**
On the Edit CASA Case page (example: https://casa-qa.herokuapp.com/casa_cases/2/edit)
Screen Shot 2020-09-26 at 4 30 47 PM (2)

**Description**
All users should see all past `court_dates` associated with a `casa_case` on the Edit CASA Case page.
All `court_dates` that have passed should become `past_court_dates` and appear here. 

**Backend**
This is part of a new feature, so an addition will need to be made to db for `past_court_dates` which can be associated with a `casa_case`.

### PR DESCRIPTION
**Pull request by matisnape**
### What github issue is this PR for, if any?
Resolves #805 

### What changed, and why?
When court date passes, a past court date is created.
Past court dates are displayed on Edit Casa Case page

### How will this affect user permissions?
- Volunteer permissions: N/A
- Supervisor permissions: N/A
- Admin permissions: N/A

### How is this tested? (please write tests!) 💖💪
I would need advice which tests should I add

### Screenshots please :)
image

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### CODE REVIEW

File: db/migrate/20201024121156_create_court_date_instance_from_casa_case_court_date_column.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      end
+    end
+  end
+
+  def down
+    CourtDate.all.each do |cd|
+      cd.destroy!
+    end
+  end
+end
```
Comment (littleforest) [inline db/migrate/20201024121156_create_court_date_instance_from_casa_case_court_date_column.rb:15]: @matisnape we are using [After Party](https://github.com/theSteveMitchell/after_party) for one-time data population tasks. After party tasks run after the data migrations run.

Comment (matisnape) [reply db/migrate/20201024121156_create_court_date_instance_from_casa_case_court_date_column.rb:15]: Yeah, I've seen it. I started with what I know and build up from there :)

**Review by compwron — APPROVED** [review]
This looks good! ✨💖
Please fix the merge conflict in `schema.rb` and then we'll merge :)

**Comment by compwron** [comment]
@matisnape I would love to merge this. Do you want help with the merge conflict?

**Comment by compwron** [comment]
I am going to merge this and then deal with it because dealing with forks is a pain

**Comment by compwron** [comment]
✨💖🙌🏼🎉

**Pull request merged**